src/main/scala/tusday.scala

```diff
@@ -0,0 +1,58 @@
+object tusday extends App{
+
+  val x = 99 to 0 by -1
+
+  def song(counter: Int):String = {
+    val counter2 = counter-1
+    s"""         |   $counter bottles of beer on the wall
+                 |   $counter bottles of beer
+                 |  take one down and pass it around, $counter2 bottles of beer on the wall """
```

Comment:
take one down and pass it around, ${counter-1} bottles of beer and then you wouldn't need the variable